### PR TITLE
Add new doc about linking OTEL apps with K8s

### PR DESCRIPTION
**Pull request by rhetoric101**
Reviewer, please check the following:

* New Kubernetes doc about how to link OTEL-instrumented apps to Kubernetes metadata.
* New doc in the OTEL collector section that points to this new Kubernetes doc.
* Check the left navigation for the Kubernetes and OTEL sections to confirm links.

### CODE REVIEW

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+  * Instrumented your applications with [OpenTelemetry](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup/), and successfully sent data to New Relic via OpenTelemetry Protocol (OTLP)
+
+If you have general questions about using collectors with New Relic, see our [Introduction to OpenTelemetry Collector with New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-intro).
+## Configure your application to send telemetry data to the OpenTelemetry collector [#instrument]
+
+To set this up, you need to add a custom snippet to the `env` stanza of your Kubernetes YAML file. We have an example below that shows the snippet for a sample front-end microservice (`Frontend.yaml`). The snippet includes two sections that do the following:
+
+  * **Section 1:** Ensure that the telemetry data is sent to the collector. This sets the environment variable OTEL_EXPORTER_OTLP_ENDPOINT with the host IP. It does this by calling the downward API to pull the host IP.
```
Comment (zuluecho9) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx:36]:
```suggestion
  * **Section 1:** Ensure that the telemetry data is sent to the collector. This sets the environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` with the host IP. It does this by calling the downward API to pull the host IP.
```

Comment (zuluecho9) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx:36]:
```suggestion
  * **Section 1:** Ensure that the telemetry data is sent to the collector. This sets the environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` with the host IP. It does this by calling the downward API to pull the host IP.
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+      exporters: [ otlp ]
+    logs:
+      receivers: [ otlp ]
+      processors: [ resourcedetection, k8sattributes, resource, batch ]
+      exporters: [ otlp ]
+```
+
+### Step 1: Configure the OTLP exporter [#otlp-exporter]
+
+First, configure it by adding an OTLP exporter to your [OpenTelemetry collector configuration YAML file](https://opentelemetry.io/docs/collector/configuration/) along with your New Relic API key as a header.
```
Comment (zuluecho9) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx:146]: Which API key are we talking here? I'm assuming license key, as that's used for ingest, but not sure. The two choices would be either license key (used for ingest) or user key (used for NerdGraph). Whatever we determine, I'd edit the variable to read something like 'YOUR_LICENSE_KEY'. And maybe also clarify in intro paragraph something like 'use the license key for the New RElic account you want to report data to'.

**Review by zuluecho9 — APPROVED** [review]

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-apm-applications-kubernetes.mdx [hunk 3]
```diff
@@ -10,6 +10,7 @@ redirects:
   - /docs/integrations/kubernetes-integration/metadata-injection/metadata-injection
   - /docs/integrations/kubernetes-integration/metadata-injection/kubernetes-apm-metadata-injection
   - /docs/kubernetes-pixie/kubernetes-integration/link-your-applications/link-your-applications-kubernetes
+  - /docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-your-applications-kubernetes
 ---
 
 You can surface Kubernetes metadata and [link it to your APM agents](/docs/integrations/kubernetes-integration/metadata-injection/kubernetes-apm-metadata-injection#agent-compatibility) as [distributed traces](/docs/understand-dependencies/distributed-tracing/get-started/introduction-distributed-tracing) to explore performance issues and troubleshoot transaction errors. For more information, see this [New Relic blog post](https://blog.newrelic.com/engineering/monitoring-application-performance-in-kubernetes/).
```
Comment (zuluecho9) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-apm-applications-kubernetes.mdx:16]:
```suggestion
You can surface Kubernetes metadata and [link it to your APM agents](/docs/integrations/kubernetes-integration/metadata-injection/kubernetes-apm-metadata-injection#agent-compatibility) as [distributed traces](/docs/understand-dependencies/distributed-tracing/get-started/introduction-distributed-tracing) to explore performance issues and troubleshoot transaction errors. For more information, see [this blog post about monitoring app performance via Kubernetes](https://blog.newrelic.com/engineering/monitoring-application-performance-in-kubernetes/).
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx]: @zuluecho9, this looks like a duplication of a previous comment, so I'm just going to close it.

File: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx]: @zuluecho9, I clarified this "ingest license key." Also, users need to use this literal environment variable name, so I clarified in the prerequisites that this is the environment variable for the ingest key.

**Review by zuluecho9 — APPROVED** [review]

**Pull request merged**